pkg/util/timeutil/BUILD.bazel

```diff
@@ -48,3 +49,48 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+# Need to check how to handle crlfmt inside generate_lowercase_timezones.go
+genrule(
+    name = "gen-lowercase-timezones-go",
+    srcs = [
+        "generate_lowercase_timezones.go",
+    ],
+    outs = ["lowercase_timezones.go"],
+    cmd = """
+      env PATH=`dirname $(location @go_sdk//:bin/go)` HOME=$(GENDIR) \
+      go run $(location generate_lowercase_timezones.go) 
```

Comment:
Is it not possible to simply do "$(location @go_sdk//:bin/go) run ..."?

Reply:
It is running either ways (with "$(location @go_sdk//:bin/go) run ..." and "go run") but it is failing due to:
`failed to run crlfmt: exec: "crlfmt": executable file not found in $PATH`

I can see `crlfmt` is kind of an internal library, right?

Reply:
It's a binary, https://github.com/cockroachdb/crlfmt. We'd have to specify it as a tool/binary dependency, in the same we declare `execgen` as a binary dependency before being allowed to use it.

Reply:
I already added inside `tools deps` (I think so):
```
    tools = [
        "@go_sdk//:bin/go",
        "@com_github_cockroachdb_crlfmt//:crlfmt",
    ],
    ```
    But I am not sure how I can add it in `$PATH` as that `crlfmt` is needed inside `generate_lowercase_timezones.go` itself (outside `genrule`)

Reply:
Perhaps we could try appending to PATH to include `($location @com_github_cockroachdb_crlfmt//:crlfmt)`?

Reply:
It is really necessary to use `go run` here? That seems like we're not leveraging Bazel for builds correctly. What I would expect is that we compile a new `go_binary` using `generate_lowercase_timezones.go`, and then have a `genrule` that just runs it. That would erase the need to depend on `@go_sdk//:bin/go`. Is there a reason we can't do this?

Reply:
I was using `go run` as it was the first thing that came to my mind (while checking the actual go:generate)

That is a good suggestion, let me try my luck on that